tests/run-make/optimization-remarks-dir-pgo/Makefile

```diff
@@ -0,0 +1,16 @@
+# needs-profiler-support
```

Comment:
It's worked around elsewhere:

```suggestion
# needs-profiler-support
# ignore-windows-gnu

# FIXME(mati865): MinGW GCC miscompiles compiler-rt profiling library but with Clang it works
# properly. Since we only have GCC on the CI ignore the test for now.
```